public/main.js

```diff
@@ -0,0 +1,37 @@
+const { app, BrowserWindow } = require('electron');
+const isDev = require('electron-is-dev');
+
+if (isDev) {
+  console.log('Running in development');
+} else {
+  console.log('Running in production');
+}
+
+let mainWindow;
+
+function createWindow() {
+  mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
```

Comment:
See #18 for some thoughts on this structure—totally good to go for now, just something to keep in mind as our app scales 🧐

Reply:
This is the window size for the entire application. Essentially this is the Parent. For siblings, not totally clear on how electron handles the sizing relative to the parent.

Reply:
Right, I guess the open question for me is whether we can use standard `vw` and `vh` units with Electron… or if we have to rely on these dimension values. If it’s the latter, just making sure we have access to them where we need them (especially since panels will expand/collapse relative to the outer-most container (i.e. the window))